linode/manager
# change: [M3-7437] - Update Profile Queries to use Query Key Factory

## Description 📝
- Introduces our custom query key factory 🏭
- Switches all profile related queries to use the new factory 🎉

## Preview 📷
![Screenshot 2024-02-20 at 12 04 08 PM](https://github.com/linode/manager/assets/115251059/09590224-ad41-4071-9524-3a63c3a6d8be)

## How to test 🧪

- Verify the result of `getQueryKeys` is type-safe
- Check the diff for any mistakes
- Verify all automated testing passes
- Use the React Query dev tools to check out the new query key structure and check for any unexpected queries

## As an Author I have considered 🤔
